src/main/java/macrobase/diagnostics/ScoreDumper.java

```diff
@@ -0,0 +1,41 @@
+package macrobase.diagnostics;
```

Comment:
Sorry I didn't get to this earlier, but ScoreDumpDiagnostic can be run as a pipeline, right? See `bin/diagnostic.sh`.